src/sentry/tsdb/snuba.py

```diff
@@ -30,11 +40,15 @@ class SnubaTSDB(BaseTSDB):
     will return empty results for unsupported models.
     """
 
-    # The ``model_query_settings`` and ``model_being_upgraded_query_settings`` are translations of
-    # TSDB models into required settings for querying snuba. Queries for ``model_columns``
-    # directly hit snuba, while queries for ``model_columns_being_upgraded`` hit
-    # redis in the main thread and snuba in a background thread.
-    model_query_settings = {
+    project_filter_model_query_settings = dict(
+        map(
+            lambda item: to_project_filter_model_query(item, snuba.Dataset.Outcomes, "times_seen"),
+            FILTER_STAT_KEYS_TO_VALUES.items(),
+        )
+    )
+
```

Comment:
you can make this more readable with a dict comprehension:

```
project_filter_model_query_setting = {
  model: to_project_filter_model_query(reason, dataset, aggregate) # of course you need to change the return of the function
  for reason, model
  in FILTER_STAT_KEYS_TO_VALUES.items()
}
```

It would make it clear what exactly you are iterating on.

Reply:
Nice! With this, I won't even need `to_project_filter_model_query`